src/usr/local/www/system_advanced_notifications.php

```diff
@@ -80,6 +80,12 @@
 ))->setHelp('When enabled, the firewall will check CA and Certificate expiration ' .
 	'times daily and file notices when expired or soon-to-expire ' .
 	'entries are detected.');
+$section->addInput(new Form_Checkbox(
+    'revoked_cert_ignore_notify',
+    'Ignore for Revoked Certificate',
```

Comment:
This is a bit longer than it needs to be, `Ignore Revoked` works better since it won't wrap. It's between two other certificate options and the checkbox description mentions certificates, so no worries about context.

Reply:
Totally agree, fixed it